File: 1985-find-the-kth-largest-integer-in-the-array/1985-find-the-kth-largest-integer-in-the-array.cpp
class Solution {
public:
    string kthLargestNumber(vector<string>& nums, int k) {
        sort(nums.rbegin(),nums.rend(),[](const string  a, const string b){
            if(a.length()==b.length()){
                return a<b;
            }
            return a.length()<b.length();
        });
        return nums[k-1];
    }
};

File: 1985-find-the-kth-largest-integer-in-the-array/1985-find-the-kth-largest-integer-in-the-array.py
class Solution(object):
    def kthLargestNumber(self, nums, k):
        """
        :type nums: List[str]
        :type k: int
        :rtype: str
        """
        cpList = [];
        nums.sort()
        for x in nums:
            cpList.append(int(x))
        cpList.sort()
        return(str(cpList[len(cpList)-k]))